contracts/mocks/FeeManagerMock.sol

```diff
@@ -12,7 +12,7 @@ contract FeeManagerMock is BaseFeeManager {
     }
 
     function getFeeManagerMode() external pure returns (bytes4) {
-        return bytes4(keccak256(abi.encodePacked("manages-one-direction")));
+        return 0xf2aed8f7; // "manages-one-direction"
```

Comment:
Please clarify in the comments how the hash was got:

// bytes4(keccak256(abi.encodePacked("manages-one-direction")));

instead of just

// "manages-one-direction"